src/RemoteHandler/ServerMock.js

```diff
@@ -221,16 +433,17 @@ class ServerMock {
             DateOfServiceIntervention:  "2016-11-02T00:00:00.000+0200",
         }];
     }
-    
+
     onReadyStateChange(callback) {
         this.onreadystatechange = callback;
     }
-    
+
     onError(callback) {
         this.onError = callback;
     }
 
     send() {
+        console.log(this.api);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AS2228)